Add more pages to scratchr2

GrahamSH-LLK opened the pull request:
This adds more pages to scratchr2.
Resolves #774 
Adds:   
		
		"https://scratch.mit.edu/login",
        "https://scratch.mit.edu/login_retry",
        "https://scratch.mit.edu/cloudmonitor/*",
        "https://scratch.mit.edu/accounts/login"

mxmou commented on addons/scratchr2/addon.json:
```diff
@@ -27,7 +27,11 @@
+        "https://scratch.mit.edu/500/",
+        "https://scratch.mit.edu/login",
+        "https://scratch.mit.edu/login_retry",
+        "https://scratch.mit.edu/cloudmonitor/*",
```
It already matches cloudmonitor (on line 27).

mxmou commented on addons/scratchr2/addon.json:
```diff
@@ -27,7 +27,11 @@
+        "https://scratch.mit.edu/login",
+        "https://scratch.mit.edu/login_retry",
+        "https://scratch.mit.edu/cloudmonitor/*",
+        "https://scratch.mit.edu/accounts/*"
```
Same, line 23.

GrahamSH-LLK commented:
@WorldLanguages There is some underlying bug/glitch. I added those because they weren't showing up.

WorldLanguages commented:
@GrahamSH-LLK Please create a bug report issue with clear reproduction steps

GrahamSH-LLK commented:
> Please create a bug report issue with clear reproduction steps

The problem is, there is none. Some page loads just are missing addons.

WorldLanguages commented:
@GrahamSH-LLK Only issue I've experienced was page loads sometimes not loading any addon if you reloaded the extension recently. I've never experienced some addons/styles being loaded, while others not

mxmou commented:
> There is some underlying bug/glitch. I added those because they weren't showing up.

For me they do show up. And if they don't, why would duplicating the pattern fix the problem?

GrahamSH-LLK commented:
> > There is some underlying bug/glitch. I added those because they weren't showing up.
> 
> For me they do show up. And if they don't, why would duplicating the pattern fix the problem?

It's not every time. I didn't realize that they were already there. I just noticed that those pages didn't have it, so I added them.

WorldLanguages commented:
Should we merge in the current state?

GrahamSH-LLK commented:
> Should we merge in the current state?

Yep!
